test/files/neg/abstract-report2.check

```diff
@@ -61,8 +61,8 @@ it has 13 unimplemented members.
 
 class Baz[T] extends Collection[T]
       ^
-abstract-report2.scala:15: error: class Dingus needs to be abstract, since:
-it has 24 unimplemented members.
+abstract-report2.scala:21: error: class Dingus needs to be abstract, since:
```

Comment:
OT/apropos of nothing, I noticed the `since:` is weird because of deprecation messaging.

Reply:
So it is. Also, why the newline?